aws/lambda/node/log-api-key-to-cloudwatch/lambda.js

```diff
@@ -29,14 +29,16 @@ exports.handler = (event, context, callback) => {
 
 	const apikey = request.headers.authorization ? request.headers.authorization[0].value : 'Anonymous';
 	const host = request.headers.host[0].value;
-	const hittype = request.headers['user-agent'] && request.headers['user-agent'][0].value == 'CL' ? 'Client Library' : 'API';
+	const useragent = request.headers['user-agent'] && request.headers['user-agent'][0].value;
```

Comment:
can we trim this to the maximum permitted characters?

Reply:
I trimmed this in the lambda where it's passed to GA, so we still log out the full  thing.